.goreleaser.yaml

```diff
@@ -11,7 +11,7 @@ builds:
       - CGO_ENABLED=1
     mod_timestamp: "{{ .CommitTimestamp }}"
     flags: &build-flags
-      - -tags=json1
+      - -tags=json1,netgo,osusergo
```

Comment:
I think if I squint I can see why osusergo is needed but it's not immediately obvious outside of opm's general domain what command is triggering the need for that tag.

/lgtm

Reply:
🙏 

I'm not _positive_ if `osusergo` is actually necessary, but since we don't have anything that actually tests the results of our binary builds, I figured it was best to err on the side of caution.